code/headers/mq_sensor.hpp

```diff
@@ -46,69 +48,49 @@ namespace r2d2::gas_detection {
          * divides it with the ro_clean_air_factor.
          * @return
          */
+        void calibrate() {
+            int val = 0;
+            for (int i = 0; i < sample_time; i++) {
+                val += resistance_calculation(adc_pin.read());
+                hwlib::wait_ms(interval_time);
+            }
+            val = (val / sample_time) / ro_clean_air_factor;
+            ro = val;
+        };            
 
-        void calibrate();
         /**
          * Gets the gas values and returns them in an array of gas_s.
          * @return
          */
-        std::array<gas_s, AmountOfGasses> get() override;
+        std::array<gas_s, AmountOfGasses> get() override {
+            int rs = 0;
```

Comment:
Same here as previous statement. two character variable names makes it hard to understand.